Ginger/GingerCoreNET/ActionsLib/ActionsConversion/BusinessFlowConversionStatus.cs

```diff
@@ -0,0 +1,190 @@
+            {
+                return mStatusIcon;
+            }
+            set
+            {
+                if (mStatusIcon != value)
+                {
+                    mStatusIcon = value;
+                    OnPropertyChanged(nameof(StatusIcon));
+                }
+            }
+        }
+
+        eImageType mSaveStatusIcon = eImageType.Pending;
+        public eImageType SaveStatusIcon
+        {
+            get
+            {
+                return mSaveStatusIcon;
+            }
+            set
+            {
+                if (mSaveStatusIcon != value)
+                {
+                    mSaveStatusIcon = value;
+                    OnPropertyChanged(nameof(SaveStatusIcon));
+                }
+            }
+        }
+
+        /// <summary>
+        /// This method is used to set the status icon
+        /// </summary>
+        private void SetStatusIcon()
+        {
+            switch (mConversionStatus)
+            {
+                case eConversionStatus.Finish:
+                    StatusIcon = eImageType.Passed;
+                    break;
+                case eConversionStatus.Stopped:
+                    StatusIcon = eImageType.Stopped;
+                    break;
+                case eConversionStatus.Pending:
+                    StatusIcon = eImageType.Pending;
+                    break;
+                case eConversionStatus.Running:
+                    StatusIcon = eImageType.Running;
+                    break;
+                case eConversionStatus.Failed:
+                    StatusIcon = eImageType.Failed;
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// This method is used to set the save status icon
+        /// </summary>
+        private void SetSaveStatusIcon()
+        {
+            switch (mSaveStatus)
+            {
+                case eConversionSaveStatus.Pending:
+                    SaveStatusIcon = eImageType.Pending;
+                    break;
+                case eConversionSaveStatus.Saved:
+                    SaveStatusIcon = eImageType.Passed;
+                    break;                
+                case eConversionSaveStatus.Saving:
+                    SaveStatusIcon = eImageType.Running;
+                    break;
+                case eConversionSaveStatus.Failed:
+                    SaveStatusIcon = eImageType.Failed;
+                    break;
+            }
+        }
+
+        bool mIsSelected = false;
+        public bool IsSelected
+        {
+            get {
+                return mIsSelected;
+            }
+            set {
+                if(mIsSelected != value)
+                {
+                    mIsSelected = value;
+                    OnPropertyChanged(nameof(IsSelected));
+                }
+            }
+        }
+
+        public string RelativeFilePath { get; set; }
+
+        public string BusinessFlowName { get; set; }
+
+        public int ActivitiesCount { get; set; }
```

Comment:
please delete all fields we do not use currently

Reply:
removed which are not required